src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml

```diff
@@ -45,14 +45,60 @@ tests:
               maxValue: 200
 
     - label: "TH reads from the DUT the (0xFFFC) FeatureMap attribute"
-      PICS: WNCV.S.Afffc
+      PICS:
+          " WNCV.S.Afffc && !WNCV.S.F00 && !WNCV.S.F01 && !WNCV.S.F02 &&
+          !WNCV.S.F03 && !WNCV.S.F04 "
       command: "readAttribute"
       attribute: "FeatureMap"
       response:
+          value: 0
           constraints:
               type: bitmap32
-              minValue: 0
-              maxValue: 32768
+
+    - label: "Given WNCV.S.F00(LF) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F00
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    - label: "Given  WNCV.S.F01(TL) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F01
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x2]
+
+    - label: "Given WNCV.S.F02(PA_LF) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F02
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x4]
+
+    - label: "Given WNCV.S.F03(ABS) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F03
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x8]
+
+    - label: "Given WNCV.S.F04(PA_TL) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F04
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x10]
```

Comment:
```suggestion
              hasMasksSet: [0x12]
```

Reply:
PA_TL must have TL set as well

Reply:
@jmeg-sfy We are checking each bit in a different step as different PICS are applicable. line no 67 checks for WNCV.S.F01(TL) flag and line no 94 checks for WNCV.S.F04(PA_TL)
![image](https://user-images.githubusercontent.com/102359958/187842683-4a03a0c1-ce6f-43e4-b431-50868ccbf530.png)